### ISSUE TITLE
[feature] mentions request: /github subscribe owner/repo mentions

### ISSUE THREAD

**Issue by campbellssource** [integrations/slack#1093]
I'd love to get notifications in slack only when I am mentioned in a given repo.

So the request is that a `[feature]` is added called mentions. So that I can type  `/github subscribe owner/repo mentions` and only get a notification in slack when someone mentions me.

**Comment by campbellssource** [comment]
It occurred to me that `/github subscribe owner/repo mentions` does not specify a use's mentions to subscribe to. This might be a bit odd in a public channel.
I really want to subscribe to mentions in the app/bot rather than a channel.

Perhaps the query could be `/github subscribe owner/repo mentions:campbellssource`

**Comment by amitkumariiit** [comment]
Thanks @campbellssource for reaching out to us and suggesting a feature. We would look into this and prioritise accordingly.

**Comment by 0xdevalias** [comment]
Related APIs for this:

- https://docs.github.com/en/rest/reference/activity#notifications
  - `mention`: You were specifically @mentioned in the content.
  - `team_mention`: You were on a team that was mentioned.

**Comment by askreet** [comment]
We would definitely make use of this. We have a custom Hubot plugin currently doing this that I'd love to toss in the bin for first-party support.

**Comment by danrademacher** [comment]
Agreed. this would be great! We're trying to use @-mentions in Slack so folks can filter out a feed of just the things they need to see. This is especially helpful for non-developerswho spend less time in Github but work on issues across many repos within our org. They don't want to sift through all the unrelated noise in full org feed, but they get mentioned across so many repos that subscribing to each one eventually just recreates most of the full org noise, with errors when folks forget to subscribe to a repo they are lightly involved with

**Comment by csaba-kovacs** [comment]
Any update on this feature?

**Comment by JolandaNaafs** [comment]
@amitkumariiit, can you look into this? We would be thrilled if it would work as described by @campbellssource.
